### DISCUSSION TITLE
blocked client, does not start graphic

### DISCUSSION THREAD

**Discussion by jumabape** [ltsp/ltsp#717] [Q&A]
Hello, I'm sorry for my horrible English but I need help, I've been trying this for a week now and I can't get anything, my project is an LTSP server with several images, so I downloaded thinstation and created my image, but when it came to implementing it at ltsp server and if it starts it gets stuck, I don't know if I do something wrong if you could help me please

first

![image](https://user-images.githubusercontent.com/106191110/170124737-c5e75b8d-9943-4e08-b5d1-edee1c9ac4b2.png)

and

![image](https://user-images.githubusercontent.com/106191110/170124698-d0db5e9a-30df-41d9-acaf-2c4bcf61bef1.png)

**Comment by alkisg** [comment]
Hi, LTSP currently supports debian-based distributions.
To netboot thinstation you'll need to add in ltsp.ipxe an appropriate kernel command line that LTSP doesn't don't know; you should ask its developers about it.
For example, to netboot ubuntu-desktop.iso these ltsp.ipxe entries are needed: https://github.com/ltsp/ltsp/wiki/Non-LTSP-iPXE-entries

In other words, if you're using thinstation you're not using LTSP (the injected ltsp.img code) at all, even though you'll be using dnsmasq, ipxe and nfs to boot it.

**Reply by jumabape** [reply]
hello, I am using LTSP as a server in ubuntu 20.01 I followed the ltsp installation steps and everything is correct, now I will try what you told me to see if it works for me.

**Reply by jumabape** [reply]
I don't understand why my image appears to me that it doesn't have vmlinuz or initrd inside, I tried a simple iso and if it works and starts

![image](https://user-images.githubusercontent.com/106191110/170266643-9a586c68-60de-4333-951c-42afcdf0929f.png)



and it does not work, Do you know another site, to create images or if you would have any to pass me, it is as I say it is for a project
